lib/server.js

```diff
@@ -9,24 +9,19 @@ class SimplyImitatedSQSHttpServer {
     this.server = null
     this.hostPort = null
     this.waiters = []
+    this.pendingItems = 0
+    this.sendItems = 0
```

Comment:
```suggestion
    this.sentItems = 0
```

It looks like this would be past tense

Reply:
willfix.